extension/parquet/parquet_reader.cpp

```diff
@@ -431,8 +456,27 @@ bool ParquetReader::PreparePageBuffers(ParquetReaderScanState &state, idx_t col_
 		col_data.payload.ptr = col_data.decompressed_buf.ptr;
 		break;
 	}
-	default:
-		throw FormatException("Unsupported compression codec. Try uncompressed, gzip or snappy");
+	default: {
```

Comment:
Parquet has 
`std::ostream& operator<<(std::ostream& out, const CompressionCodec::type& val) {`
for this

Reply:
Good point, using it now ^^